### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by SergeyTeslov**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @loponavt

### CODE REVIEW

**Comment by d-makarenko** [comment]
Небольшое замечание:
если R < 2, то в значении функции выводится "не число". Непонятно, что значит "не число", хотелось бы там видеть "значение функции не определено"
```
y(-10,0) = не число
y(-9,8) = не число
y(-9,6) = не число
y(-9,4) = не число
y(-9,2) = не число
y(-9,0) = не число
```

**Comment by SergeyTeslov** [comment]
Оценка: 5/5

File: Homework2/HomeWork2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                    if (x>(-8-r)&&x<(-8+r))
+                    {
+                        Console.WriteLine("y({0:0.0}) = {1:0.00}", x, segment1(x, r));
+                    }
+                    else
+                    {
+                        Console.WriteLine("значение функции не определено");
+                    }
+                }
+                else if (x < 2)
```
Comment (SergeyTeslov) [inline Homework2/HomeWork2/Program.cs:50]: в точке x = 2 необходимо было указать два значения функции

File: Homework2/HomeWork2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        {
+            Console.WriteLine("Введите значение R - радиуса окружности: ");
+            double r = double.Parse(Console.ReadLine());
+            if (r < 2)
+            {
+                Console.WriteLine("Введенное значение R приведет к тому, что функция будет иметь область неопределенных значений");
+            }
+            for (double x = -10; x <= 8.01; x += 0.2)
+            {
+                if (x < -6)
```
Comment (SergeyTeslov) [inline Homework2/HomeWork2/Program.cs:39]: в точке x= -6 необходимо было указать два значения функции при условии что радиус больше 2 
если радиус равен 0, то необходимо было указать, что в точке x= -8 y(x)=-2

**Review by SergeyTeslov — COMMENTED** [review]
image
при r=1 что то непонятное с областью корректности y(-8.8,-7.0) при том что центр окружности в -8

**Pull request merged**